0000-deb-repo-additional-headers.md

```diff
@@ -88,6 +88,50 @@ The integrity of the `deb` binary packages must be verified using the checksums
 
 This will be handled in a separate RFC.
 
+## Failure Registry
+
+Packages might have sync issues, such as wrong GPG signature or
+checksum mismatch or downloading issues etc. However, broken packages
+still can be assigned to the channels/repos and thus potentially
+invaliadate them. In some circumstances this might have serious impact
```

Comment:
I think a broken package is / should not be imported. But such an error means that the repo is "incomplete" which
can cause dependency problems when somebody try to use it,

Reply:
We don't have to though. But I would say, better we do, so we can also examine it later at "low level" using tools like `spacewalk-data-fsck` and sort of things. Also such error in general will cause deps problems _anyway_, regardless we have those "leftovers" from the package in place on disk or not. In any case, we still need a list of those broken files to be shown in UX somehow, hence the DB should at least know about the file and a cause why it was failed.

For that I would actually import it and mark as broken.